### PR TITLE
📝 Clarify access tokens

### PR DESCRIPTION
**Pull request by coyotte508**
Clarify misconceptions, eg https://github.com/huggingface/huggingface.js/issues/69#issuecomment-1460859901

### CODE REVIEW

File: packages/hub/README.md [hunk 1]
```diff
@@ -45,7 +47,7 @@ await deleteRepo({ repo, credentials });
 When uploading large files, you may want to run the `commit` calls inside a worker, to offload the sha256 computations.
 
 Also, use `Blob` to avoid loading the whole files in RAM. In `Node`, it's up to you to provide a smart `Blob` wrapper around your file. Feel free to open an issue if you want *us* to provide the smart `Blob` implementation. If you use `Bun`, `Bun.File` is already a smart blob implementation and can be used directly!
-
+P
```
Comment (radames) [inline packages/hub/README.md:50]: typo

**Review by radames — COMMENTED** [review]
thank you for the improvements
there's one typo

File: packages/hub/README.md (hunk 1 above)
Comment (coyotte508) [reply packages/hub/README.md:50]:
```suggestion

```

**Review by radames — APPROVED** [review]

**Comment by coyotte508** [comment]
Thanks for the review @radames !

**Pull request merged**